src/it/scala/org/zalando/spearheads/innkeeper/routes/PostPathsSpec.scala

```diff
@@ -52,38 +52,38 @@ class PostPathsSpec extends FunSpec with BeforeAndAfter with Matchers {
           path.hostIds should be(hostIds)
         }
       }
-    }
 
-    describe("when a token with the admin scope is provided") {
```

Comment:
Please check individual commits for this pull request. There were two specs hanging around when they should have been in the `describe("success")` block.